src/encoded/schemas/supplementary_file.json

```diff
@@ -0,0 +1,97 @@
+{
+    "title": "Supplementary File",
+    "$id": "/profiles/supplementary_file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "file_format",
+        "filename",
+        "submission_centers",
+        "submitted_id",
+        "data_type",
+        "data_category"
+    ],
+    "identifyingProperties": [
+        "accession",
+        "submitted_id",
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/description"
+        },
+        {
+            "$ref": "mixins.json#/file_links"
+        },
+        {
+            "$ref": "mixins.json#/file_release"
+        },
```

Comment:
Not sure these are needed. May want to talk with Elizabeth about whether these will need annotated filenames.